components/Content.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import "../styles/Content.css";
+
+export function Content(props)
+{
+    return (
+        <div className = "Content">
+            {props.content.split('\n').map((str) => <p key = {str}>{str}</p>)}
```

Comment:
str is not defined and we haven't find the way to fetch content from database. We'll think about the detail in the future.

Reply:
In this, `str` is a variable defined by the anonymous function passed to the `map` function. The more verbose version of this is:
```ts
  props.content.split()  // Returns a string[]
    .map(                // Map
      (str: string) => {
        return (
           <p key={str}>
             {str}       // Content passed to render
           </p>
        );
      }
    );
```

I recommend looking at [this page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) for more info on the `map` function. It's very useful!

As for database functionality, we keep that separate from the rest of the components to enforce separation of concerns. These components solely render data, so we'll have to write additional functionality to fetch and transform data from our database.